tools/cicd/integration-tests.yaml

```diff
@@ -26,13 +26,32 @@ steps:
     - PYTHON_VERSION=3.11
     - .
```

Comment:
You can probably remove these 2 first steps https://github.com/Kaggle/kagglehub/pull/65#discussion_r1455197468